tests/sqlite3/test-sqlite3.cpp

```diff
@@ -53,6 +53,62 @@ TEST_CASE("SQLite rowid", "[sqlite][rowid][oid]")
     sql << "drop table test1";
 }
 
+static void SetupForeignKeysTest(soci::session& sql)
```

Comment:
It would be better to make this a class doing this in its ctor and executing `drop table xxx` in its dtor to ensure that the tables don't stay in the database even if an exception is thrown.